.github/workflows/build-artifact.yml

```diff
@@ -0,0 +1,59 @@
+name: Build artifact
+
+on:
+  push:
+    paths:
+      - 'plugin/**'
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths:
+      - 'plugin/**'
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set artifact name
+      shell: bash
+      run: |
+        sha=$(echo ${{ github.sha }} | cut -c 1-8)
+        echo "::set-env name=artifact_sha::$sha"
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Fetch dependencies
+      run: |
+        python .\util\fetch_deps.py
+
+    - name: Set up msbuild
+      uses: microsoft/setup-msbuild@v1.0.1
+
+    - name: Add Custom Problem Matcher
+      run: |
+        echo "::add-matcher::.github/matchers/msbuild.json"
```

Comment:
Can this be:
```
echo "::add-matcher::.github/matchers/${{ github.job }}.json"
```

Reply:
Sure!
Then we should change the job name as `msbuild` too?

Reply:
Sure; either change the name to msbuild within the job or change the problem matcher to build; I think both options are viable!